### ISSUE TITLE
Microphone is greyed out, windows 10. Bluetooth headset. (works in audacity)

### ISSUE THREAD

**Issue by Innomen** [telegramdesktop/tdesktop#16281]
### Steps to reproduce
1. launch telegram, select chat, look at microphone
2. 
3. 

### Expected behaviour
clickable microphone to record a voice message

### Actual behaviour
Button grey, no function.

### Configuration
**Operating system:**
Windows 10

**Version of Telegram Desktop:**
2.7.4 x64

**Installation source (Linux Only)** - the official website / GitHub releases / flatpak / snap / distribution package:

**Used theme**:
default

<details><summary><b>Logs</b>:</summary>
Insert log.txt here (if necessary)
</details>

**Comment by ilya-fedin** [comment]
> Button grey, no function.

Well, it's always grey, but you should hold it rather than click.

**Comment by Aokromes** [comment]
> ### Steps to reproduce
> 1. launch telegram, select chat, look at microphone
> 
> ### Expected behaviour
> clickable microphone to record a voice message
> 
> ### Actual behaviour
> Button grey, no function.
> 
> ### Configuration
> **Operating system:**
> Windows 10
> 
> **Version of Telegram Desktop:**
> 2.7.4 x64
> 
> **Installation source (Linux Only)** - the official website / GitHub releases / flatpak / snap / distribution package:
> 
> **Used theme**:
> default
> 
> Logs:

check on calls if you have the mic properly selected.

**Comment by Aokromes** [comment]
> ### Steps to reproduce
> 1. launch telegram, select chat, look at microphone
> 
> ### Expected behaviour
> clickable microphone to record a voice message
> 
> ### Actual behaviour
> Button grey, no function.
> 
> ### Configuration
> **Operating system:**
> Windows 10
> 
> **Version of Telegram Desktop:**
> 2.7.4 x64
> 
> **Installation source (Linux Only)** - the official website / GitHub releases / flatpak / snap / distribution package:
> 
> **Used theme**:
> default
> 
> Logs:

p.s record voice message is push to talk, not push and talk.

**Comment by ilya-fedin** [comment]
It always playback via system default device. If you sure you have the right device configured as default device, then we need Telegram log to see what OpenAL actually chooses.

**Comment by Innomen** [comment]
I think this is the correct log. And yeah looking at it it's apparently trying the speakers. I'm trying to use my bluetooth headset. But I don't know, I didn't read it top to bottom. Plus I don't know what I'm looking at really.

[log.txt](https://github.com/telegramdesktop/tdesktop/files/6446810/log.txt)

**Comment by ilya-fedin** [comment]
Maybe it's just not set as the default in Windows settings?

**Comment by Innomen** [comment]
Ok here's a more recent log and you can see the headset stuff.

[log (2).txt](https://github.com/telegramdesktop/tdesktop/files/6447631/log.2.txt)

**Comment by ilya-fedin** [comment]
the log says your default playback device is still "Realtek(R) Audio"

**Comment by Innomen** [comment]
Well it's not by all accounts, I changed it every way accessible from the call settings menu and in system sound settings. Can you tell me what I'm supposed to do step by step in case I'm missing something? ELI5 X)

**Comment by ilya-fedin** [comment]
Can you show the sound settings dialog (the Windows 7 one, not UWP one)?

**Comment by Innomen** [comment]
![image](https://user-images.githubusercontent.com/134324/117614594-8739e200-b136-11eb-9c6b-3cd591b36fc4.png)

Is this the one you mean?

**Comment by Aokromes** [comment]
check recording and then https://i.imgur.com/DA56Itt.png

**Comment by ilya-fedin** [comment]
> Is this the one you mean?

Yeah, really weird, the log says you have Realtek Audio as default... But it also says that it's from 2021.05.04... Are you sure you sending actual logs?

**Comment by ilya-fedin** [comment]
btw two log files seem to be identical
